### PR TITLE
Cleanup pkg.py

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

File: rpmlint/pkg.py [hunk 1]
```diff
@@ -511,25 +511,25 @@ def _extract(self):
             print_warning('Unable to access dir %s' % self.dirname)
             return None
```
Comment (scarabeusiv) [inline rpmlint/pkg.py]: There is return cmd bit down removed so this one should not return either.

**Comment by scarabeusiv** [comment]
@mimi1vx https://lgtm.com/projects/g/rpm-software-management/rpmlint/rev/pr-d440e4457c1b92c8077be31a8b33a1df3cce51ae

**Comment by scarabeusiv** [comment]
./rpmlint/pkg.py:522:13: F841 local variable 'cmd' is assigned to but never used

**Comment by mimi1vx** [comment]
> ./rpmlint/pkg.py:522:13: F841 local variable 'cmd' is assigned to but never used

in progress with removal of `getstatusoutput`

**Pull request merged**